### PR TITLE
Update language for aliasing `node_modules/.bin`

### PR DESCRIPTION
**Pull request by zachhale**
To keep consistent with [builder-react-app](https://github.com/FormidableLabs/builder-react-app/pull/1)

### CODE REVIEW

**Comment by boygirl** [comment]
LGTM!

**Comment by zachhale** [comment]
I'll wait for the other PR to be approved before merging. Thanks!

File: dev/DEVELOPMENT.md [hunk 1]
```diff
@@ -4,16 +4,14 @@ Development
 We use [builder][] and `npm` to control all aspects of development and
 publishing.
 
-As a preliminary matter, please update your shell to include
+As a preliminary matter so you can type `builder` instead of
+`./node_modules/.bin/builder` for all commands, please update your shell to include
 `./node_modules/.bin` in `PATH` like:
 
 ```sh
 $ export PATH="${PATH}:./node_modules/.bin"
```
Comment (ryan-roemer) [inline dev/DEVELOPMENT.md]: Haha -- it's different here. Do whatever you're doing in other PR here too :wink:

**Comment by ryan-roemer** [comment]
@zachhale -- One comment, rest LGTM without re-review.

**Pull request merged**